### PR TITLE
[Core] Small clean up EliminationBS

### PR DESCRIPTION
**Pull request by loumalouomega**
Part of #3245

### CODE REVIEW

File: kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h [hunk 1]
```diff
@@ -1251,7 +1251,7 @@ class ResidualBasedEliminationBuilderAndSolver
                     #pragma omp atomic
                     r +=  v;
 #else
-                    values_vector[pos] += Alocal(i_local,j);
+                    values_vector[pos] += rALocal(i_local,j);
```
Comment (philbucher) [inline kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h:1254]: I think this should also be changed somewhere else

Comment (loumalouomega) [reply kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h:1254]: What do you mean?

Comment (philbucher) [reply kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h:1254]: this is the only place where this is changed
I am pretty sure this also needs to be changed elsewhere

Comment (loumalouomega) [reply kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h:1254]: I am cleaning the class step by step, as #3245 changed much code at the same time

Comment (philbucher) [reply kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h:1254]: ok now I see

**Review by philbucher — COMMENTED** [review]
ok from my side

**Comment by loumalouomega** [comment]
Ping @RiccardoRossi

**Comment by loumalouomega** [comment]
> Ping @RiccardoRossi

Ping @RiccardoRossi

File: kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h (hunk 1 above)
Comment (rubenzorrilla) [reply kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h:1254]: I think this is a GitHub misbehaviour (I checked the master version and this is correct).

**Review by rubenzorrilla — APPROVED** [review]

**Pull request merged**